templates/nfs-exports.j2

```diff
@@ -1,11 +1,9 @@
 {% if bootstrap is defined %}
-/data           {{ bootstrap.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
+/export           {{ bootstrap.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
 {% endif %}
 {% for m in masters %}
-/data           {{ m.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
-/data/registry  {{ m.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
+/export           {{ m.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
 {% endfor %}
 {% for w in workers %}
-/data           {{ w.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
-/data/registry  {{ w.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
+/export           {{ w.ipaddr }}(rw,sync,no_wdelay,no_root_squash,insecure)
```

Comment:
This file looks out of date... as it's already been fixed.
https://github.com/redhat-cop/ocp4-helpernode/blob/main/templates/nfs-exports.j2

please create a new branch with only the vars/main.yml and templates/named.conf.j2 changes.